mirdata/rwc_popular.py

```diff
@@ -0,0 +1,374 @@
+    url='https://staff.aist.go.jp/m.goto/RWC-MDB/AIST-Annotation/AIST.RWC-MDB-P-2001.CHORD.zip',
+    checksum='68379c88bc8ec3f1907b32a3579197c5'
+)
+ANNOTATIONS_REMOTE_4 = utils.RemoteFileMetadata(
+    filename='AIST.RWC-MDB-P-2001.VOCA_INST.zip',
+    url='https://staff.aist.go.jp/m.goto/RWC-MDB/AIST-Annotation/AIST.RWC-MDB-P-2001.VOCA_INST.zip',
+    checksum='47ded648a496407ef49dba9c8bf80e87'
+)
+
+
+class Track(object):
+    def __init__(self, track_id, data_home=None):
+        if track_id not in INDEX:
+            raise ValueError('{} is not a valid track ID in RWC-Popular'.format(track_id))
+
+        self.track_id = track_id
+        self._data_home = data_home
+        self._track_paths = INDEX[track_id]
+
+        if METADATA is None or METADATA['data_home'] != data_home:
+            _reload_metadata(data_home)
+
+        if track_id in METADATA.keys():
+            self._track_metadata = METADATA[track_id]
+        else:
+            # annotations with missing metadata
+            self._track_metadata = {
+                'piece_number': None,
+                'suffix': None,
+                'track_number': None,
+                'title': None,
+                'artist': None,
+                'singer_information': None,
+                'track_duration_sec': None,
+                'instruments': None,
+                'drum_information': None,
+            }
+
+        self.audio_path = utils.get_local_path(
+            self._data_home, self._track_paths['audio'][0])
+
+        self.piece_number = self._track_metadata['piece_number']
+        self.suffix = self._track_metadata['suffix']
+        self.track_number = self._track_metadata['track_number']
+        self.title = self._track_metadata['title']
+        self.artist = self._track_metadata['artist']
+        self.singer_information = self._track_metadata['singer_information']
+        self.track_duration_sec = self._track_metadata['track_duration_sec']
+        self.instruments = self._track_metadata['instruments']
+        self.drum_information = self._track_metadata['drum_information']
+
+    @utils.cached_property
+    def sections(self):
+        return _load_sections(utils.get_local_path(
+                self._data_home, self._track_paths['sections'][0]))
+
+    @utils.cached_property
+    def beats(self):
+        return _load_beats(utils.get_local_path(
+                self._data_home, self._track_paths['beats'][0]))
+
+    @utils.cached_property
+    def chords(self):
+        return _load_chords(utils.get_local_path(
+                self._data_home, self._track_paths['chords'][0]))
+
+    @utils.cached_property
+    def vocal_instrument_activity(self):
+        return _load_voca_inst(utils.get_local_path(
+                self._data_home, self._track_paths['voca_inst'][0]))
+
+
+def download(data_home=None, force_overwrite=False):
+    save_path = utils.get_save_path(data_home)
+    dataset_path = os.path.join(save_path, DATASET_DIR, 'annotations')
```

Comment:
dataset path should just be `mir_datasets/rwc_popular`. Maybe call this variable `annotations_path`?